.golangci.yml

```diff
@@ -90,6 +90,15 @@ linters:
         - name: var-declaration
         - name: var-naming
 
+formatters:
+  enable:
+    - gofmt
+    - goimports
+  settings:
+    goimports:
+      local-prefixes:
+        - github.com/mgechev/revive
```

Comment:
I would recommend keeping gci.

goimports doesn't enforce the block of imports people are used to.

I would use goimports without settings, and gci with the ones you had